jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/TestPlanStats.java

```diff
@@ -136,6 +136,145 @@
      */
     double sentBytesPerSecond();
 
+    /**
+     * Gets the duration from jmeter send request to jmeter
+     * get the first batch of response (like TTFB).
```

Comment:
maybe makes sense to put a clarification on TTFB meaning? maybe (like time to first byte, aka: TTFB)

Reply:
batch -> byte